lib/hanami/events/subscriber.rb

```diff
@@ -2,12 +2,12 @@ module Hanami
   module Events
     class Subscriber
       def initialize(pattern, block)
-        @pattern = pattern
+        @pattern = Matcher.new(pattern)
```

Comment:
Could be renamed to `@pattern_matcher` since in this case that variable holds instance of matcher, not a pattern.